src/test/fuzz/float.cpp

```diff
@@ -19,7 +19,7 @@ FUZZ_TARGET(float)
 
     {
         const double d{[&] {
-            double tmp;
+            double tmp{0};
```

Comment:
Not sure if we want to maim code to please broken and ancient compilers

Reply:
This only change is better than disabling a useful warning, isn't it?

Reply:
No. C++ allows for uninitialized values, and allows for valgrind to find code logic error based on this.

I fail to see how using a warning emitted by a known-broken compiler is an excuse to worsen the code.

Either the compiler shouldn't be used or the warning be disabled on the compiler.

Reply:
Looks like this (and other changes) can be dropped after #27013